apps/grafana/CHANGELOG.md

```diff
@@ -8,11 +8,18 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
+## [1.1.0] - 2022-04-06
```

Comment:
A very tiny detail : do we need to update the release date ?

Reply:
Sure we can!

Reply:
I find it weird when you say "bump grafana release" with a :bookmark: gitmoji.
Either it's potsie and we are making a release, or it's grafana and we are making a dependency upgrade?

Reply:
I will fix this as "bump grafana tray release". Is it more clear?